components/vars/css/components/spectrum-actionbutton.css

```diff
@@ -1835,26 +1820,25 @@
   --spectrum-actionbutton-xl-quiet-emphasized-textonly-border-size-selected-disabled: var(--spectrum-alias-border-size-thin);
   --spectrum-actionbutton-xl-quiet-emphasized-textonly-hold-icon-color-selected-disabled: var(--spectrum-alias-icon-color-disabled);
   --spectrum-actionbutton-xl-quiet-emphasized-textonly-background-color-selected-disabled: var(--spectrum-global-color-gray-200);
-  --spectrum-actionbutton-xl-quiet-emphasized-textonly-border-color-selected-disabled: var(--spectrum-alias-border-color-disabled);
+  --spectrum-actionbutton-xl-quiet-emphasized-textonly-border-color-selected-disabled: var(--spectrum-alias-border-color-transparent);
```

Comment:
ah I see, you are avoiding adding more tokens and just say it's `        "spectrum-actionbutton-l-quiet-emphasized-background-color-disabled": "transparent",`

Reply:
This is auto-generated, but yeah!